perf/impl/quic-go/v0.34/Makefile

```diff
@@ -10,7 +10,7 @@ perf-${commitSha}: perf-${commitSha}.zip
 	unzip -o perf-${commitSha}.zip
 
 perf-${commitSha}.zip:
-	wget -O $@ "https://github.com/quic-go/perf/archive/${commitSha}.zip"
+	wget -O $@ "https://github.com/mxinden/perf/archive/${commitSha}.zip"
```

Comment:
I just merged https://github.com/quic-go/perf/pull/12, let's update the PR here.